.github/workflows/release-desktop.yml

```
@@ -115,7 +115,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: "16.x"
```

Maybe v18 could be used as it has a longer support window (until 2025).
```suggestion
          node-version: "18.x"
```